#goal definition
string group_name
string ik_frame

---
#result definition

# Note: The expected priority order of the error should match the message order
uint16 UNKNOWN=720
uint16 TIMEOUT=721
uint16 CALCULATION_FAIL=722
uint16 INVALID_INPUT=723
uint16 NONE=0

uint16 error_code_id

geometry_msgs/Pose pose
---
#feedback definition
